pkg/reconciler/mtbroker/delivery_config_test.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2021 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mtbroker
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/utils/pointer"
+
+	eventingduck "knative.dev/eventing/pkg/apis/duck/v1"
+
+	. "knative.dev/pkg/configmap/testing"
```

Comment:
Wherever possible it would be nice to avoid `.` imports. Even something short such as `cmt` or `cmtesting` helps with readability I think.